scripts/ai/AIDriveStrategyUnloadCombine.lua

```diff
@@ -592,8 +989,19 @@ function AIDriveStrategyUnloadCombine:onLastWaypointPassed()
     elseif self.state == self.states.BACKING_UP_FOR_REVERSING_COMBINE then
         self:setNewState(self.stateAfterMovedOutOfWay)
         self:startRememberedCourse()
+    elseif self.state == self.states.MOVING_BACK_BEFORE_PATHFINDING then
+        if self.state.properties.pathfinderController then
+            self:debug('Retry last pathfinding after moved back a bit')
+            self:setNewState(self.states.WAITING_FOR_PATHFINDER)
+            self.lastState.properties.pathfinderController:retry(self.lastState.properties.pathfinderContext)
```

Comment:
I like the idea of this. If we want to use this more often, than a more generic idea would be great :)